### ISSUE TITLE
Adding tabula sapiens data to a reference panel?

### ISSUE THREAD

**Issue by hyunhwan-jeong** [GuangshengPei/deCS#1]
Hi @GuangshengPei,

CZI has published their tabula sapiens database - https://tabula-sapiens-portal.ds.czbiohub.org/celltypes, and I wonder you can add the data to deCS. I believe the very enriched resource is beneficial.

Best regards,

Hyun-Hwan Jeong

**Comment by GuangshengPei** [comment]
Hi，Hwan. Thanks for your suggestion. I will try to integrate it to deCS in
the future.
Is this is mouse reference?

Hyun-Hwan Jeong ***@***.***> 于2021年4月6日周二 下午1:25写道：

**Comment by hyunhwan-jeong** [comment]
No, it would be a human reference. You can find a mouse reference from https://tabula-muris.ds.czbiohub.org.

Best,

Hyun-Hwan Jeong
